Orange/classification/base_classification.py

```diff
@@ -18,27 +16,8 @@ class ModelClassification(Model):
 
 
 class SklModelClassification(SklModel, ModelClassification):
-    def predict(self, X):
-        prediction = super().predict(X)
-        if not isinstance(prediction, tuple):
-            return prediction
-        values, probs = prediction
-
-        class_vars = self.domain.class_vars
-        max_values = max(len(cv.values) for cv in class_vars)
-        if max_values == probs.shape[-1]:
-            return values, probs
-
-        if not self.supports_multiclass:
-            probs = probs[:, np.newaxis, :]
-        probs_ext = np.zeros((len(probs), len(class_vars), max_values))
-        for c, used_vals in enumerate(self.used_vals):
-            for i, cv in enumerate(used_vals):
-                probs_ext[:, c, cv] = probs[:, c, i]
-        if not self.supports_multiclass:
-            probs_ext = probs_ext[:, 0, :]
-        return values, probs_ext
+    pass
 
 
 class SklLearnerClassification(SklLearner, LearnerClassification):
-    __returns__ = SklModelClassification
+    __returns__ = SklModel
```

Comment:
Why was this changed?

Reply:
It is my mistake. First I wanted to get rid of SklLearnerClassification class, but then I decided to do refactoring in separate (following) PR. It would be too many changes for this PR.